internal/oscap/oscap.go

```diff
@@ -35,6 +38,9 @@ const (
 	defaultCentos9Datastream string = "/usr/share/xml/scap/ssg/content/ssg-cs9-ds.xml"
 	defaultRHEL8Datastream   string = "/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml"
 	defaultRHEL9Datastream   string = "/usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml"
+
+	// autotailor directory path
+	autotailorDirPath string = "/usr/share/osbuild"
```

Comment:
I'm no expert on SCAP, but wouldn't it make sense to use something like `/usr/share/xml/scap/osbuild` to make it obvious what is this directory for? :thinking:

Reply:
I would check with the OSCAP team...

Reply:
Waiting to hear back on this.

Reply:
I'm not sure what to suggest. The `autotailor` utility can store it's output to any file path and the `oscap` command accepts any file path in the `--tailoring-file` option, so in theory, you can put the file anywhere.

But, the path `/usr/share/xml/scap` is owned by the `scap-security-guide` RPM package, and I'm afraid that it isn't a good practice to add files to paths owned by some package, so I would rather avoid saving this there. From this point of view, the `/usr/share/osbuild` seems a better solution.

We had a similar situation in OSCAP Anaconda Addon, which is our plugin in the Anaconda installer. There, we put the artifacts to `/root/openscap-data` so that users can review them easily after the installation finishes.

Reply:
Thanks for the input @jan-cerny

Reply:
> Based on FHS (https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html) and the discussion, I suggest to put the tailoring file into a subdirectory under /usr/share/xml, e.g. /usr/share/xml/osbuild-openscap-data. This makes it explicit that the file was created by osbuild, but it is not related to osbuild as the tool in any way...

Awesome, I like this suggestion `/usr/share/xml/osbuild-openscap-data` and I like, as you mentioned, that it makes it explicit that it was created by osbuild.